language/classify_text/README.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
```

Comment:
This should (ideally) use an autogenerated readme. This readme has way too much content and possibly duplicates the tutorial on devsite.

Reply:
done.